<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Cell Planning Tools Settings</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="general">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="tab1MainLayout">
       <item>
        <widget class="QLabel" name="lbSectorLayer">
         <property name="text">
          <string>Cell-sector Layer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QgsMapLayerComboBox" name="cbSectorLayer">
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbDEMList">
         <property name="text">
          <string>Digital Elevation Model (DEM) raster list</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hlDem">
         <item>
          <widget class="QLineEdit" name="leDem">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>0 inputs selected</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbDem">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="lbDSMList">
         <property name="text">
          <string>Digital Surface Model (DSMraster list</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="leDsm">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>0 inputs selected</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbDsm">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="lbSensibility">
         <property name="text">
          <string>DEM/DSM Sensibility</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="sbSensibility"/>
       </item>
       <item>
        <widget class="QLabel" name="lbSilobeLimit">
         <property name="text">
          <string>Upper Sidelobe Distance limit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbSidelobLimit">
         <property name="maximum">
          <number>100000</number>
         </property>
         <property name="value">
          <number>30000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbUnits">
         <property name="text">
          <string>Units</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbUnits">
         <item>
          <property name="text">
           <string>Meters</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Feet</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>516</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
